### PR TITLE
Add bb test files

### PR DESCRIPTION
**Pull request by nvictus**

### CODE REVIEW

**Review by jackh726 — COMMENTED** [review]
The bigBedExample file is already in the pybigtools test data directory - should move instead of adding a new file.

What is the other file for?

File: pybigtools/tests/test_api.py [hunk 1]
```diff
@@ -245,6 +248,18 @@ def test_zoom_records(bw, bb):
     assert pytest.raises(KeyError, bb.zoom_records, 3911, "chr11")
 
 
+# def test_values_no_end(bw, bb):
```
[the remark this answers is not in the document]
Comment (nvictus) [reply pybigtools/tests/test_api.py:251]: It's currently failing (panic). Was gonna address in a separate PR.

**Comment by nvictus** [comment]
> What is the other file for?

An example bigBed file with a custom autoSql schema (genePred).

**Comment by jackh726** [comment]
I've identified the issue with the added test

**Comment by jackh726** [comment]
Okay, both issues are fixed - can you rebase?

File: bigtools/resources/test/bigBedExample.bb
Comment (jackh726) [inline bigtools/resources/test/bigBedExample.bb:1]: Can you delete this since it is already in the repo?

**Pull request merged**